Lib9c/Battle/RankingSimulator.cs

```diff
@@ -19,19 +19,26 @@ public class RankingSimulator : Simulator
         private readonly EnemyPlayer _enemyPlayer;
         private readonly int _stageId;
         private List<ItemBase> _reward;
+        private readonly ArenaInfo _arenaInfo;
+        private readonly ArenaInfo _enemyInfo;
+        private readonly AvatarState _avatarState;
         public override IEnumerable<ItemBase> Reward => _reward;
 
-        public RankingSimulator(
-            IRandom random,
+        public RankingSimulator(IRandom random,
```

Comment:
Just for consistency :)
```suggestion
        public RankingSimulator(
            IRandom random,
```